packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -1,6 +1,7 @@
 openapi: 3.0.3
 info:
   title: PDND Interoperability API
+  description: PDND Interoperability API Gateway for machine-to-machine communication
```

Suggestion to make this more specific
```suggestion
  description: PDND Interoperability API for machine-to-machine interaction with interoperability features
```